home/ivp-planning/advanced-tools/virtual-cameras.md

```diff
@@ -29,12 +29,12 @@ Alternatively, it is also possible to create a new virtual camera that will posi
 Virtual cameras behave like regular objects when it comes to [position ](../machines/move-objects.md)and [rotation](../machines/scale-and-rotate-objects.md). As other objects, they appear in the 2D- and 3D-panel and can be [moved around](../machines/move-objects.md) or [rotated](../machines/scale-and-rotate-objects.md) in the same way.
 
 {% hint style="info" %}
-It is also possible to [move ](../machines/move-objects.md)or [rotate](../machines/scale-and-rotate-objects.md) several cameras at once. To do so, simply [select ](../machines/selecting-and-moving-objects.md)multiple cameras before you make changes in the [2D- ](../user-interface/the-2d-panel.md)or [3D-panel](../user-interface/the-3d-panel.md).
+It is also possible to [move](../machines/move-objects.md)or [rotate](../machines/scale-and-rotate-objects.md) several cameras at once. To do so, simply [select](../machines/selecting-and-moving-objects.md)multiple cameras before you make changes in the [2D-](../user-interface/the-2d-panel.md)or [3D-panel](../user-interface/the-3d-panel.md).
```

Comment:
You have removed the white spaces from the links, but did not add them back into the text in most cases. Please make sure to insert them everywhere where necessary.

```suggestion
It is also possible to [move](../machines/move-objects.md)or [rotate](../machines/scale-and-rotate-objects.md) several cameras at once. To do so, simply [select](../machines/selecting-and-moving-objects.md) multiple cameras before you make changes in the [2D-](../user-interface/the-2d-panel.md) or [3D-panel](../user-interface/the-3d-panel.md).
```